README.md

```diff
@@ -168,3 +168,19 @@ module.exports = {
   ]
 }
 ```
+
+### `path: 'path/to/files'`
```

Comment:
Sorry for the long delay.

A small suggestion: change the option from `path` to `outputPath`.

Then we are good to go!